src/components/Control/Control.jsx

```diff
@@ -0,0 +1,18 @@
+export const Control = ({
+  filterBy,
+}) => (
+  <div
+    className="control"
+
+  >
```

Comment:
If you have a small number of props (less than 3) and they can fit comfortably on the same line without causing readability issues, you don't necessarily need to wrap each prop to a new line. Keeping them on the same line can make the code more compact and easier to read.
```suggestion
  <div className="control">
```